diagnostic.md

```diff
@@ -53,9 +66,9 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+//class = "big"; id = "contentPane"; class = "important red"; <div><p></p></div>;
 
-<!-- your answer ends here -->
+.big {...}; #contentPane {...}; .important .red {...}; div p {...};
```

Comment:
`.big` and `#contentPane` are correct. 

`.important.red` would apply styling to elements that have BOTH .important and .red styling. When you have the space, it says applying the same styling to both .important and .red classes. Big difference!

`div > p` the '>' is the notation for parent to child relationship.